### ISSUE TITLE
[BUG] Re-adding person to group has no effect

### ISSUE THREAD

**Issue by fabrei** [nanu-c/axolotl#338]
### Description

Re-adding a member to group, which changed the phone, but not the phone number, does not work. 

### Steps to Reproduce

1. Go into group
2. Click the burger button
3. Choose "Edit group"
4. Choose "+ Members"
5. Choose the member in contact list
6. Click on "x"
7. Choose "Update group"

**Expected behavior:**
Get an information about updated the group and the member was re-added to the group and receives messages.

**Actual behavior:** 
I get the information that the group was updated and the person is now in the group, but the person isn't. The person does not get any message from the group.

#### Versions

ut 16.04 ota 15
axolotl v0.9.5

#### Device

oneplusone bacon

#### Link to Debug Log
[application-click-textsecure.nanuc_textsecure_0.9.5.log](https://github.com/nanu-c/axolotl/files/5897771/application-click-textsecure.nanuc_textsecure_0.9.5.log)

I changed the phone number of the person to add into "<person_to_add>" and other members of this group to "<another_member_of_group>".

**Comment by Flaburgan** [comment]
Hello @fabrei and thank you to take time to report an issue on Axolotl. Due to upstream changes in Signal, many things related to contacts and groups management are broken right now, sorry about that. Unfortunately, they changed many things and the fixes aren't trivial. @nanu-c is working a lot on it but we can't say when this will be solved. If you want to know more or give an hand you can join our [Telegram channel](https://t.me/axolotl_dev).

**Comment by fabrei** [comment]
Ja, I already noticed in some of your issues, that there are major changes in the upstream of signal. so, no pressure from my side in fixing this specific issue. and thanks for the link, i joined now =)
